integration/main.server.ts

```diff
@@ -1 +1,12 @@
+import { enableProdMode } from '@angular/core';
+
+import { environment } from './environments/environment';
+
+if (environment.production) {
+  enableProdMode();
+}
+
 export { AppServerModule } from './app/app.server.module';
+export { ngExpressEngine } from '@nguniversal/express-engine';
+
+export { renderModule, renderModuleFactory } from '@angular/platform-server';

```

Comment:
What is the reason for these exports and running the `enableProdMode();` in this file now?
Also, some of these exports are not used and can be removed.

Reply:
yes, you are right, enableProd not needed
but without the following imports, Angular SSR does not start

```ts
export { AppServerModule } from './app/app.server.module';

export { renderModule, renderModuleFactory } from '@angular/platform-server';
```